content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -502,7 +502,7 @@ Operators may add or remove APIs at any time - always specify an `olm.gvk` depen
 
 ### Set a minimum version
 
-The Kubernetes [documentation on API changes](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#readme) describes what changes are allowed for kube-style operator APIs. These versioning conventions allow an operator to update an API without bumping the `apiVersion`, as long as the API is backwards-compatible.
```

Comment:
HTML proofer was complaining about this one and I think it is legit, so I fixed it too.